website/docs/core-concepts/building-ui/dynamic-ui/README.md

```diff
@@ -19,11 +19,11 @@ Every property of a widget can be described dynamically using JavaScript inside
 
 ## Updating widget data
 
-Let us take an example of a [table displaying ](../../../reference/widgets/table/#table-data)a list of products. When a user selects a product in the table, we may want to update the product information in a form so that the user can update the product.
+Let us take an example of a [table displaying ](/reference/widgets/table/#table-data)a list of products. When a user selects a product in the table, we may want to update the product information in a form so that the user can update the product.
```

Comment:
```suggestion
Let us take an example of a [table displaying](/reference/widgets/table#table-data) a list of products. When a user selects a product in the table, we may want to update the product information in a form so that the user can update the product.
```